.github/workflows/phpunit-mysql.yml

```diff
@@ -123,7 +114,6 @@ jobs:
         run: |
           mkdir data
           ./occ maintenance:install --verbose --database=mysql --database-name=nextcloud --database-host=127.0.0.1 --database-port=$DB_PORT --database-user=root --database-pass=rootpassword --admin-user admin --admin-pass admin
-          ./occ app:enable --force user_ldap
```

Comment:
revert

Reply:
Thank you, I got confused in my file management and forgot to revert this one :+1: